src/main/java/org/symphonyoss/symphony/messageml/elements/TableCell.java

```diff
@@ -27,11 +28,27 @@
  */
 public class TableCell extends Element {
   public static final String MESSAGEML_TAG = "td";
+  private static final String ATTR_ROWSPAN = "rowspan";
+  private static final String ATTR_COLSPAN = "colspan";
 
   public TableCell(Element parent) {
     super(parent, MESSAGEML_TAG);
   }
 
+  @Override
+  void buildAttribute(org.w3c.dom.Node item) throws InvalidInputException {
+    switch (item.getNodeName()) {
+      case ATTR_ROWSPAN:
+        setAttribute(ATTR_ROWSPAN, getStringAttribute(item));
```

Comment:
We should validate that the spans are numberrs and non-negative. You can do it here by using _getLongAttribute()_ (or, better still, adding a method _getIntAttribute()_) or  in the  _validate()_ method which you'd have to override.

Reply:
fixed. Please check now